zap/src/test/resources/org/zaproxy/zap/spider/parser/html/InputElementsSpiderHtmlParser.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+<base href="http://example.com/sample/">
+<meta charset="UTF-8">
+<title>Input Elements - Spider HTML Parser</title>
+</head>
+<body>
+<!--  Ignored: -->
+<input>
+
+<!--  src -->
+<input type="image" src="/relative/src" name="test" value="test">
```

Comment:
With a leading slash wouldn't this be an absolute path?

(Ex: Server root would be the leading slash.)